argus/backend/service/build_system_monitor.py

```diff
@@ -68,17 +73,42 @@ def check_filter(self, group_name: str) -> bool:
 
 
 class JenkinsMonitor(ArgusTestsMonitor):
+
+    BUILD_SYSTEM_FILTERED_PREFIXES = [
+        "byo",
+        "releng",
+        "load-test",
+        "sct_repair",
```

Comment:
why are those being filtered ?

Reply:
I don't quite remember why, it's better to ask @roydahan about it.